Update Weibull AFT Notebook

AmitKus opened the pull request:
1) Fixed: AttributeError: module 'statsmodels' has no attribute 'datasets'
2) Fixed: FutureWarnings
3) Updated the notebook to follow the consistent style


**Thank your for opening a PR!**

Depending on what your PR does, here are a few things you might want to address in the description:
+ [ ] what are the (breaking) changes that this PR makes?
+ [ ] important background, or details about the implementation
+ [ ] are the changes—especially new features—covered by tests and docstrings?
+ [ ] consider adding/updating relevant example notebooks
+ [ ] right before it's ready to merge, mention the PR in the RELEASE-NOTES.md

AmitKus committed:
1) Fixed: AttributeError: module 'statsmodels' has no attribute 'datasets'
2) Fixed: FutureWarnings
3) Updated the notebook to follow the consistent style

AmitKus committed:
1) Fixed: AttributeError: module 'statsmodels' has no attribute 'datasets'
2) Fixed: FutureWarnings
3) Updated the notebook to follow the consistent style

aloctavodia committed:
SMC-ABC add distance, refactor and update notebook (#3996)

* update notebook

* move dist functions out of simulator class

* fix docstring

* add warning and test for automatic selection of sort sum_stat when using wassertein and energy distances

* update release notes

* fix typo

* add sim_data test

* update and add tests

* update and add tests

tirthasheshpatel committed:
add docs for interpretation of length scales in periodic kernel (#3989)

* fix the expression of periodic kernel

* revert change and add doc

* FIXUP: add suggested doc string

* FIXUP: revertchanges in .gitignore

AlexAndorra commented:
Thanks a lot for your first PR @AmitKus 🎉 
However, it seems like no change was recorded -- when I go to ReviewNB, it tells me there are no changes to the notebook. Are you sure you commited and pushed your changes?

AmitKus committed:
1) Fixed: AttributeError: module 'statsmodels' has no attribute 'datasets'wq
2) Fixed: FutureWarnings
3) Updated the notebook to follow the consistent style

AmitKus committed:
Merge branch 'update-doc' of https://github.com/AmitKus/pymc3 into update-doc

AmitKus commented:
> Thanks a lot for your first PR @AmitKus tada
> However, it seems like no change was recorded -- when I go to ReviewNB, it tells me there are no changes to the notebook. Are you sure you commited and pushed your changes?

Even after trying hard not to mess up my first PR, I ended up commiting to the wrong branch! I have pulled in the changes now but looks like there are others also commiting in the PR due to my bad initial title (thanks for changing it). Would you recommend closing this PR and resubmitting a fresh one to just have my changes as the build is breaking now? Sorry for the trouble. I am still trying to wrap my head around all this.

AlexAndorra commented:
Ah indeed, there was some mix up with your branch -- you picked up some other commits that aren't supposed to be there.
Yeah, the easier is probably to close this PR and open a new one -- create a new branch based off master, so that you have the latest stable changes, then work and commit on this branch, and open the PR when you're ready. [This tutorial](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#changing-the-branch-range-and-destination-repository) should be helpful.
And no worries, the git workflow is powerful but hard -- kind of like black magic 😉 🧙‍♂️
